src/tools/tidy/src/issues.txt

```diff
@@ -1370,7 +1370,6 @@
 "ui/issues/issue-70093/issue-70093.rs",
 "ui/issues/issue-77218/issue-77218-2.rs",
 "ui/issues/issue-77218/issue-77218.rs",
-"ui/issues/issue-100605.rs",
```

Comment:
Should the new name of the test be added to this list?

Reply:
The top of the file says "⚠️⚠️⚠️NOTHING SHOULD EVER BE ADDED TO THIS LIST⚠️⚠️⚠️" :) 

Long-term we want all tests named "issue-..." to be renamed to something better. This PR takes care of one of the many files.

Reply:
Ah, I get it. So this is a list of files that still need to be renamed. Sounds good!